experiments/opencog/question2atomese/src/main/java/org/opencog/vqa/QuestionToOpencogApp.java

```diff
@@ -110,11 +110,33 @@ private static void closeStream(Closeable stream, Closeable dflt) {
     private static void handleException(Exception e) {
         e.printStackTrace();
     }
+    
+    public static String getHeader() {
```

Comment:
I would say the header is the part of QuestionRecord not QuestionToOpencogApp, because question record knows actual format